packages/icon/README.md

```diff
@@ -83,14 +85,51 @@ const SomeComponent = () => (
 We also export the default icon set for you! If you want to include our glyphs with your custom glyphs, you can do something like this:
 
 ```js
-import { createIconComponent, glyphs } from '@leafygreen-ui/Icon';
+import { createGlyphComponent, createIconComponent } from '@leafygreen-ui/icon';
 
 const myGlyphs = {
   ...glyphs,
-  MyCustomGlyph: () => <svg />,
+  MyCustomGlyph: createGlyphComponent('MyCustomGlyph', () => <svg />),
```

Comment:
Should we recommend spreading props onto the `<svg>` element within their custom icon?